.travis.yml

```diff
@@ -13,6 +13,16 @@ matrix:
   allow_failures:
     - rust: nightly
   fast_finish: true
+install:
+  - |
+    if [[ "$TRAVIS_RUST_VERSION" = "1.25.0" ]]; then
+      rustup target add aarch64-apple-ios armv7-apple-ios armv7s-apple-ios x86_64-apple-ios i386-apple-ios &&
+      # Cargo-lipo needs to be Beta version available on master and not crates version to handle building in subdirectories
+      cargo install --git https://github.com/TimNN/cargo-lipo &&
```

Comment:
Is it possible to pin this version to a SHA?  We shouldn't be fetching something random off the Web, basically ever.